scripts/katana_requirements/__main__.py

```diff
@@ -199,8 +199,16 @@ def get_apt_version():
     version_information_str = str(subprocess.check_output(["apt-get", "-v"], stderr=subprocess.DEVNULL), "UTF-8")
     version_match = re.match(r"apt ([0-9.]+) \(.*\)", version_information_str)
     if not version_match:
-        raise RuntimeError(f"apt-get returned unexpected version information:\n{version_information_str}")
-    return Version(version_match.group(1))
+        # Try matching against ubuntu tagged versions instead
+        version_match = re.match(r"apt ([0-9.]+)ubuntu([0-9.]+) \(.*\)", version_information_str)
+        if not version_match:
+            raise RuntimeError(f"apt-get returned unexpected version information:\n{version_information_str}")
+
+        version_string = version_match.group(1).split("u")[0]
+    else:
+        version_string = version_match.group(1)
```

Comment:
I think there regexs can be combined into something like this:

```
pt ([0-9.]+)(?:ubuntu[0-9.]+)? \(.*\)
```
(`(?:` begins a non-capturing group)

Though I'm not sure why you are splitting on `"u"` below. I think that might be a left over from an initial attempt.

Reply:
that worked, thanks.